packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata.service.ts

```diff
@@ -367,6 +367,31 @@ export class ObjectMetadataService extends TypeOrmQueryService<ObjectMetadataEnt
 
     const updatedObject = await super.updateOne(input.id, input.update);
 
+    if (input.update.isActive !== undefined) {
```

Comment:
Could we extract this to a private function?

Reply:
Sure, thank you for the feedback, we will work on it

Reply:
I had to force push because of a conflict error.